sdk/attestation/attestation/test/public/AttestationTests.spec.ts

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert, use as chaiUse } from "chai";
+import chaiPromises from "chai-as-promised";
+chaiUse(chaiPromises);
+
+import { Recorder } from "@azure/test-utils-recorder";
+
+import { createRecordedClient } from "../utils/recordedClient";
+import { AttestationClient } from "../../src";
+import { decodeString } from "../utils/base64";
+
+describe("[AAD] Attestation Client", function() {
+  let recorder: Recorder;
+  let client: AttestationClient;
+
+  beforeEach(function() {
+    // eslint-disable-next-line no-invalid-this
+    ({ client, recorder } = createRecordedClient(this, "AAD"));
+  });
+
+  afterEach(async function() {
+    await recorder.stop();
+  });
+
+  const _runtimeData =
+    "wFdC6gBMrrej2JTuNlTjWOe-ebL7Rz34WjmEUnbfFEc_5BITs2t4V8uuEI8JX73t0g_nUTu6g07xyC6rx9wl8IUQFYyP" +
+    "KhsMk3FLESkryhb5dz9cDxoxwMNnGbu-B7AsOBCe3lckQmoRAEf4_5qUm-PS26DD3SkbNRT-XjMQMQ19Q33dpKFvXPrQ" +
+    "yvCK0ly0pL-JXXdnT4hsJUn8tJKW152W2gZWeXIKO8Ge2er_8xXUvQ6gCLZwwcD1--Whg90h9n5tVRNQdqCnWwsFL0LE" +
+    "KVNiCj7Cbii8_XpjYjTTSQKSOiC_i_VbZZF9cY4W_1ZpUj7WWkSSkPhNSuqBHOvmuFrVTlfQvgdsKYQ5zYbSnPtqJ1_4" +
+    "QUoPJsYQIxyFFncIDbuGWuTPd_FDKLBLQADyO4kYWjnVMXdM1p_xjtqo2_UWTznEfrQpoZttQE99GZVEVSXPBn0GXzph" +
+    "4JDKyWq3rDIvzFMhumG5ay1eyQ622hxwBN4WVxVjJ-BtaWMnU15o4OZZVReCpTodGZabT0RgAmJqKNZnH_Vx_ECLKxss" +
+    "xEHoNWZBUCWAS9Qy4OpdQZ1-vINHJaTIZsehSZrkk1a5ttJdghTSUJGbEPWt3Azstjidyq8x1l5q-PIClhJE_Q_vHOvT" +
+    "zxCebqZOhFJl08rx8I2OYxzekLA1miJ4aZs8h3eB6tOHZF06gJC8wcIORvy8d8ysEZvja40AWSg";
+
+  const _sgxQuote =
```

Comment:
Comments explaining what's in these will be helpful for future visitors.

Reply:
FWIW, this is an SGX Quote generated from an SQL Server SGX enclave. It describes the contents of the enclave and can be used to verify that the enclave is actually the correct enclave created by the SGX team.